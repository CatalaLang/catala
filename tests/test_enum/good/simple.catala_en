## Article

```catala
declaration enumeration E:
  -- Case1 content integer
  -- Case2

declaration scope A:
  context x content E
  context y content integer

scope A:
  definition x equals Case1 content 2
  definition y equals match x with pattern
    -- Case1 of i : 42
    -- Case2 : 43
```
