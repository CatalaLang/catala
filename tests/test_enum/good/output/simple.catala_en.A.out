[RESULT] Computation successful! Results:
[RESULT] x = Case1 2
[RESULT] y = 42
